extensions/amp-ad-network-nws-impl/0.1/nws-a4a-config.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {startsWith} from '../../../src/string';
+
+export function nwsIsA4AEnabled(win, element, useRemoteHtml) {
```

Comment:
is this used?

Reply:
No, but I was following this checklist: https://github.com/ampproject/amphtml/blob/master/ads/google/a4a/docs/Network-Impl-Guide.md#create-the-configuration-file

Reply:
I think because you are just always returning `true` in `_a4a.config.js` this is not needed. The idea is that you could have more complex logic here if you wanted. Lets delete it.

Reply:
Sure, removed.